PWGDQ/TableProducer/tableMaker.cxx

```diff
@@ -182,12 +182,13 @@ struct TableMaker {
   struct : ConfigurableGroup {
     Configurable<bool> fConfigRunZorro{"cfgRunZorro", false, "Enable event selection with zorro [WARNING: under debug, do not enable!]"};
     Configurable<string> fConfigZorroTrigMask{"cfgZorroTriggerMask", "fDiMuon", "DQ Trigger masks: fSingleE,fLMeeIMR,fLMeeHMR,fDiElectron,fSingleMuLow,fSingleMuHigh,fDiMuon"};
+    Configurable<bool> fConfigRunZorroSel{"cfgRunZorroSel", false, "Select events with trigger mask"};
```

Comment:
Is this option needed? We have one already at L183 but I see that that one is just tagging the events and not really filtering them. We should discuss this with Luca @lucamicheletti